[js] remove global generator installation (#915)
SAP/cloud-sdk: docs-js/features/odata/generate-odata-client.mdx

npm install -D @sap-cloud-sdk/generator
```

:::note

> We recommend not to use a globally installed generator, because this hides the used generator version and causes problems for the transpilation to JavaScript code.

Reviewer: [pp]

```suggestion
We recommend to install the generator as a local dependency, because global installations hide the used generator version and cause problems when transpiling to JavaScript.
```

> If you must use a globally installed generator, you have to install the`@types/node` and `@sap-cloud-sdk/odata-v2` or `@sap-cloud-sdk/odata-v4` package in your project to make the transpilation work.

Reviewer: [nth]

```suggestion
If you must use a globally installed generator, install the`@types/node` and `@sap-cloud-sdk/odata-v2` or `@sap-cloud-sdk/odata-v4` package in your project to make the transpilation work.
```
